provider-blackduck/src/main/java/com/synopsys/integration/alert/provider/blackduck/issue/BlackDuckProviderIssueHandler.java

```diff
@@ -73,11 +75,10 @@ public void createOrUpdateBlackDuckIssue(BlackDuckProviderIssueModel issueModel,
     }
 
     private void performRequest(HttpUrl httpUrl, HttpMethod httpMethod, IssueRequest issueRequest) throws IntegrationException {
-        Request request = new BlackDuckRequestBuilder(new Request.Builder())
-                              .url(httpUrl)
+        BlackDuckResponseRequest request = new BlackDuckRequestBuilder()
                               .method(httpMethod)
-                              .bodyContent(new StringBodyContent(gson.toJson(issueRequest)))
-                              .build();
+                              .bodyContent(new StringBodyContent(gson.toJson(issueRequest), BodyContentConverter.DEFAULT))
```

Comment:
I'm surprised there's no constructor with this default converter.

Reply:
Me too.